package.json

```diff
@@ -8,7 +8,8 @@
   "scripts": {
     "start": "nodemon --exec \"probot\" run ./index.js",
     "serve": "probot run ./index.js",
-    "test": "jest && codecov && standard"
+    "test": "jest && codecov && standard",
+    "postinstall": "cp -R node_modules/probot probot"
```

Comment:
Does this work on Windows?
I think it'd be neater if we had this put into ./.serverless

Reply:
No, no it does not. I believe if we tried messing with the Webpack `include` setting it might do the trick. Currently its set to include `__dirname`, I wonder if we could just give it `[__dirname, 'node_modules/probot']`.

Reply:
Oooh or we could try changing the `require()` in `handler.js` to `require('node_modules/probot')`